service/routes.py

```diff
@@ -162,3 +161,27 @@ def check_content_type(media_type):
         status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
         "Content-Type must be {}".format(media_type),
     )
+
+######################################################################
+#  Increment Interested Counter
+######################################################################
+
+
+@app.route('/recommendations/<int:rec_id>/interested', methods=["PUT"])
```

Comment:
As the counter keeps increasing, would this perhaps be a POST request?

Reply:
Yes, I agree. I guess I made a mistake here. PUT is better for a status changing action. I have revised the code.